coord/src/module.c

```diff
@@ -2386,6 +2366,11 @@ static int initSearchCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int
   MR_Init(cl, clusterConfig.timeoutMS);
   InitGlobalSearchCluster(clusterConfig.numPartitions, slotTable, tableSize);
 
+  if (clusterConfig.type == ClusterType_RedisOSS) {
+    // Initiate the topology updater after the MR cluster is initialized
+    InitRedisTopologyUpdater(ctx);
```

Comment:
I think we still want to do it only on `ClusterType_RedisOSS`

Reply:
Ohh you do it only in this case :+1:

Reply:
and we do :)